DataScience/LogDownloader.py

```diff
@@ -176,7 +178,10 @@ def download_container(app_id, log_dir, container=None, conn_string=None, accoun
 
             try:
                 bp = bbs.get_blob_properties(container, blob.name)
-
+                # If total downloaded file size + current file size has exceeded max_size then stop downloading more
+                Logger.info('Total downloaded cooked log size in MB : {}\n'.format(total_download_size_MB))
+                if((total_download_size_MB + bp.properties.content_length/(1024**2)) > max_size):
+                    break
```

Comment:
nit: Worth a log when we stop downloading early? I could see needing to answer a question from a customer about why they don't see evaluation results for the full time range they select.

Reply:
Added a log here